### DISCUSSION TITLE
[help asap] SEB failed to completely lockdown computer

### DISCUSSION THREAD

**Discussion by KFSVFDRVER** [SafeExamBrowser/seb-win-refactoring#239] [Q&A]
hello, i have mentti app and try to take my exam unproctored, but i cannot launch it huhu, SEB keeps on saying _**failed to completely lockdown computer**_, please help

**Comment by KFSVFDRVER** [comment]
@danschlet hello hope you can help me :'(

**Reply by danschlet** [reply]
No, @dbuechel  is responsible for the Windows version.

**Comment by dbuechel** [comment]
Can you please provide more information on what exactly is happening resp. not working, ideally with the logfiles (to be found under `%LocalAppData%\SafeExamBrowser\Logs`) and a printscreen / photo of the error message.
